content/hi/kubernetes.md

```diff
@@ -0,0 +1,43 @@
+---
+title: कुबेरनेट्स
+status: पूरा हुआ
+category: तकनीकी
+tags: ["infrastructure", "", ""]
+---
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक खुला स्रोत कंटेनर ऑर्केस्ट्रेटर है जो
+आधुनिक अवसंरचना पर [कंटेनराइझड़](/container/) अनुप्रयोगों के जीवन-चक्र को स्वचालित करता है।
+यह एक डेटा सेंटर ऑपरेटिंग सिस्टम की तरह है जो एक [वितरित प्रणाली] (/distributed-systems/) पर चल रहे अनुप्रयोगों का प्रबंधन करता है
+(ठीक आपके लैपटॉप के OS की तरह जो आपके ऐप्स को प्रबंधित करता है)। 
+
+कुबेरनेट्स एक [क्लस्टर] (/cluster/) में [नोड्स] (/nodes/) में कंटेनरों को शेड्यूल करता है। 
+यह कई बुनियादी ढांचे के निर्माण को बंडल करता है, जिसे कभी-कभी "आदिम" कहा जाता है।
+एक ऐप के उदाहरण की तरह, बैलेंसर्स लोड करें, लगातार स्टोरेज, और अन्य एक साथ
+इस तरह से कि उन्हें अनुप्रयोगों में बनाया जा सकता है। 
```

Comment:
`बैलेंसर्स लोड करें, लगातार स्टोरेज, और अन्य एक साथ इस तरह से कि उन्हें अनुप्रयोगों में बनाया जा सकता है। `
load balancers, persistent storage are technical terms so we localise it not translate it.